roles/edpm_bootstrap/tasks/update_package.yml

```diff
@@ -0,0 +1,21 @@
+---
+# Copyright 2023 Red Hat, Inc.
+# All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+- name: Perform system update
+  become: true
+  ansible.builtin.dnf:
+    name: "*"
+    state: latest # noqa: package-latest
```

Comment:
```
- name: Perform system update
  become: true
  ansible.builtin.dnf:
    name: "*"
    state: latest  # noqa: package-latest
  register: edpm_bootstrap_system_update
  until: edpm_bootstrap_system_update.rc == 0
  retries: "{{ edpm_bootstrap_download_retries }}"
  delay: "{{ edpm_bootstrap_download_delay }}"
```

Reply:
and, in `download_cache.yml` task file (from `edpm_bootstrap` role) add:

```
- name: Download updated packages
  become: true
  ansible.builtin.dnf:
    name: "*"
    state: latest  # noqa: package-latest
    download_only: true
  register: edpm_bootstrap_system_updates_download
  until: edpm_bootstrap_system_updates_download.rc == 0
  retries: "{{ edpm_bootstrap_download_retries }}"
  delay: "{{ edpm_bootstrap_download_delay }}"
```

Reply:
Thank you @rebtoor :-)

Reply:
@raukadah can you add a `update_cache: true` to both dnf tasks please?